docker/Dockerfile

```diff
@@ -120,7 +120,7 @@ RUN BOOST_VERSION=1.75.0 && \
 # Install Google Benchmark support library
 ENV BENCHMARK_DIR=/opt/benchmark
 RUN SCRATCH_DIR=/scratch && mkdir -p ${SCRATCH_DIR} && cd ${SCRATCH_DIR} && \
-    git clone https://github.com/google/benchmark.git -b v1.4.1 && \
+    git clone https://github.com/google/benchmark.git -b v1.6.1 && \
```

Comment:
Why did you need to update?

Reply:
Because version 1.4 does not compile with newer version of gcc

Reply:
Not sure I understand. Does it compile with an older version of gcc, but not with a newer?

Reply:
Yes, because they forgot some includes of the standard library. It happens to work in older version of gcc but in the newer ones, you get an error about missing include